### ISSUE TITLE
BatchNorm Layer incompatible with BVLC

### ISSUE THREAD

**Issue by amirgholami** [intel/caffe#234]
Dear Whom It May Concern,

When trying to export a model trained using Intel Caffe, we get a compatibility error due to the different implementation of BN layer in IntelCaffe. The specific error is as follows:

Check failed: target_blobs.size() == source_layer.blobs_size() (3 vs. 5) Incompatible number of blobs for layer BatchNorm1

Is it possible to add a compatible version with BVLC caffe? This would allow us to convert a model trained using IntelCaffe to pytorch and other frameworks as well

Many Thanks,
-Amir

**Comment by ftian1** [comment]
you can have a try my repo https://github.com/ftian1/intelcaffe_weight_to_bvlc
